.github/workflows/ci-test-node.js.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os:
+        - ubuntu-latest
+        - windows-latest
+        - macos-latest
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+        # TODO: test on Node.js 17 (+) *after* upgrade to Webpack v5
+        node-version:
+        - 12.x
```

Comment:
I think this should test back to Node.js 10 due to this: https://github.com/brodybits/rollup-plugin-size-snapshot/blob/e62caff1e6a0d3fb8dee91d7dc246f6143629c0a/package.json#L74

```
  "engines": {
    "node": ">=10",
```

and I wonder if it would work any further back (??)